packages/viz/src/lib/style/helpers/color-bins-style.ts

```diff
@@ -5,27 +5,55 @@ import {
   findIndexForBinBuckets
 } from './utils';
 
-import { Classifier } from '../../utils/Classifier';
+import { Classifier, ClassificationMethod } from '../../utils/Classifier';
 import { Style } from '../Style';
 import {
   CartoStylingError,
   stylingErrorTypes
 } from '../../errors/styling-error';
-import { ColorBinsStyleOptions, defaultColorBinsStyleOptions } from '..';
 import { StyledLayer } from '../layer-style';
-import { toDeckStyles } from './style-transform';
 import { NumericFieldStats, GeometryType } from '../../sources/Source';
+import { getStyleValue, BasicOptionsStyle, getStyles } from '../default-styles';
+
+export interface ColorBinsOptionsStyle extends Partial<BasicOptionsStyle> {
+  // Number of size classes (bins) for map. Default is 5.
+  bins: number;
+  // Classification method of data: "quantiles", "equal", "stdev". Default is "quantiles".
+  method: ClassificationMethod;
+  // Assign manual class break values.
+  breaks: number[];
+  // Palette that can be a named cartocolor palette or other valid color palette.
+  palette: string[] | string;
+  // Color applied to features which the attribute value is null.
+  nullColor: string;
+  // Color applied to features which the attribute value is not in the breaks.
+  othersColor: string;
+}
+
+function defaultOptions(
+  geometryType: GeometryType,
+  options: Partial<ColorBinsOptionsStyle>
+): ColorBinsOptionsStyle {
+  return {
+    bins: 5,
+    method: 'quantiles',
+    breaks: [],
+    palette: getStyleValue('palette', geometryType, options),
```

Comment:
These are the params that I think should be in this file, and not in defaults. On the other hand, I think they probably don't change much depending on the geometry type

Reply:
If we do this, the pallete style should be defined in 3 files ColorBins, ColorCategories, ColorContinuous

Reply:
Why don't we expose the palette (which doesn't depend on geometryType) as a const in that file?

```js
palette: DEFAULT_QUANTITATIVE_PALETTE
```